client/components/paginate/PaginatorControlGroup.tsx

```diff
@@ -1,7 +1,12 @@
 import React from "react";
-import PropTypes from "prop-types";
 
-const PaginatorControlGroup = ({ callback, children, label }) => {
+interface PaginatorControlGroupProps {
+    callback: React.MouseEventHandler<HTMLButtonElement> | undefined;
```

Comment:
Why is undefined allowed?

Reply:
In javascript, a function without a return type returns undefined. I'll change it to void since typescript offers that type.

Reply:
Makes sense; thanks for the clarification!